source/reference/collection-level-filtering.txt

```diff
@@ -17,6 +17,62 @@ Filtered Sync
 
 .. include:: /includes/api/facts/filtering-intro.rst
 
+Starting in 1.1, ``mongosync`` supports inclusion filters to specify which
+databases and collections to include in sync.  Starting in 1.6, ``mongosync``
+also supports exclusion filters and regular expressions.
+
+- With inclusion filters, ``mongosync`` syncs matching
+  databases and collections.
+- With exclusion filters, ``mongosync`` syncs all databases
+  and collections, except for those that match the filters.
+- With both inclusion and exclusion filters, ``mongosync`` only syncs
+  databases and collections that match the inclusion filters.   It ignores
+  the exclusion filters.
+- With no filters, ``mongosync`` syncs all databases and collections.
+
+.. _c2c-filter-syntax:
+
+Filter Syntax
+-------------
+
+The :ref:`c2c-api-start` API endpoint accepts two fields that configure 
+filtered sync: ``includeNamespaces`` and ``excludeNamespaces``.
+Each field takes an array of filters that specify the databases and collections 
+to include or exclude from sync.
+
+.. note::
+
+   If the :ref:`c2c-api-start` call uses both ``includeNamespaces`` and 
+   ``excludeNamespaces`` parameters, ``mongosync`` uses the inclusion 
+   filters and ignores the exclusion filters.
```

Comment:
I think there's a misunderstanding here - the exclusion filter should override the inclusion filter - it's not ignored.

Reply:
Fixed